library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt

```diff
@@ -58,15 +58,15 @@ class SwipeToDismissHandler(
                 }
                 return true
             }
-            MotionEvent.ACTION_MOVE                          -> {
+            MotionEvent.ACTION_MOVE -> {
                 if (isTracking) {
                     val translationY = event.y - startY
                     swipeView.translationY = translationY
                     onSwipeViewMove(translationY, translationLimit)
                 }
                 return true
             }
-            else                                             -> {
+            else -> {
```

Comment:
I do not understand why we have so many changes.
The plan was to change the rules one by one.
So:
- create the .editorconfig with the current rules, eventually format the whole code base to fix existing issues.
- Change the rules one by one, in .editorconfig and then format, so for instance, change the rule for `arrow alignment` and see in the PR only changes about the arrows.

WDYT?

Reply:
The format here is actually without any rule changes. The problem we're facing is the app doesn't consistently abide by our currently set rules and breaks many basic lint rules in lots of places

Reply:
The current rule is to align the arrows in a `when` block vertically and here it is changing, so I do not understand.

Reply:
(Updated the PR fixing this. This file no longer has any diffs)